functions-home.php

```diff
@@ -0,0 +1,457 @@
+<?php
+/**
+ * Homepage functions
+ */
+
+function hide_editor_from_homepage() {
```

Comment:
Possible make this function a bit dynamic so you can reuse it inside home_meta_boxes() ?  Rather than copying the same block of code again.